routers/private/remove_track.py

```
@@ -0,0 +1,80 @@
+    edit_text_message = get_edit_text_message(callback_message)
+
+    state_data = await state.get_data()
+    playlist_name = state_data["playlist_to_remove_track"]
+
+    user_id = get_user_id(callback)
+    track_index = int(callback_text.split(":")[1])
+
+    user_db_id = ps.get_user_id(user_id)
+
+    success = ps.remove_track_by_index(user_db_id, playlist_name, track_index)
+    if success == True:
```

Ruff (0.12.2)

Avoid equality comparisons to `True`; use `success:` for truth checks

Replace with `success`

(E712)